### PR TITLE
add Triplet Sum in Array

### PR DESCRIPTION
**Pull request by soumikbaithalu**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes, I have read it.

### Description

adding the file in NeoAlgo\Python\other\TripletSumValue.py

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#6495

### CODE REVIEW

File: Python/other/TripletSumValue.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+INPUT:
+
+Enter the elements: 1 4 45 6 10 8 
+Enter K: 13
+
+OUTPUT:
+Array which sum 13  is  [(1, 4, 8)]
+
+'''
```
Comment (bhav09) [inline Python/other/TripletSumValue.py]: Add a line at EOF

File: Python/other/TripletSumValue.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+Given an array arr and an integer k.Find if there's a triplet in the array which sums up to the given integer k
+Input:
```
Comment (bhav09) [inline Python/other/TripletSumValue.py]: Need not write Sample IO at the beginning of the file. It has to be there only once and that too after the code.

**Review by bhav09 — CHANGES_REQUESTED** [review]
Reviewed.

File: Python/other/TripletSumValue.py (hunk 1 above)
Comment (soumikbaithalu) [reply Python/other/TripletSumValue.py]: @bhav09 Done

File: Python/other/TripletSumValue.py (hunk 2 above)
Comment (soumikbaithalu) [reply Python/other/TripletSumValue.py]: okay I will not write

Comment (soumikbaithalu) [reply Python/other/TripletSumValue.py]: @bhav09, please check my modified file

**Pull request merged**